_layouts/index.html

```diff
@@ -0,0 +1,65 @@
+---
+layout: base
+---
+
+
+<!-- style="background-color: blue;" -->
+  <div class="container content {{ layout.css }} {{ page.css }}">
+        <!-- section1 START-->
+        <div class="columns section1 has-text-centered-mobile is-vcentered is-flex" >
+            <div class="column section1_text">
+                <p class="hallo_text">Hallo! Everybody!</p>
+                <h1 class="is-size-3-mobile welcome_text">Welcome to our online & interactive game
+                    on DNA data analysis</h1>
+                <p class="come_join_text">Learn Bioinformatics online through interesting games! Come join us now!</p>
+                <form action="{{ site.baseurl }}/game/intro/">   
+                    <button type="submit">
+                        Get Started
+                    </button>
```

Comment:
Why a form here and not just a button?
Btw, the button become white when hovered, making it difficult to see